db/migrate/20230808110739_create_foods.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFoods < ActiveRecord::Migration[7.0]
+  def change
+    create_table :foods do |t|
+      t.string :name
+      t.string :measurement_unit
+      t.decimal :price
+      t.integer :quantity
+      t.references :user, null: false, foreign_key: true
+
```

Comment:
As part of the project requirement, you are expected to add indexes on all migrations where you know that query time might get too much (recipe and recipe_foods). This will help query time be optimized.

Reply:
@DuaneDave I appreciate the comment on this. However, if you look at the schema.rb file in the DB, the indexes were created by the rails generate commands. It would be ineffective to add another index as the ones provided are sufficient. Kindly verify this.